js/sia.js

```diff
@@ -16,12 +16,12 @@ const nodePath = require('path')
 function SiadWrapper () {
   // siad details with default values
   var settings = {
-    fileName: process.platform === 'win32' ? 'siad.exe' : 'siad',
     detached: false,
     address: 'localhost:9980',
     rpcAddress: ':9981',
     hostAddress: ':9982',
-    path: nodePath.join(__dirname, '..', 'Sia')
+    datadir: nodePath.join(__dirname, '../Sia'),
+    path: nodePath.join(__dirname, '../Sia/', process.platform === 'win32' ? 'siad.exe' : 'siad')
```

Comment:
I'm pretty sure that, at least pedantically, you want it to be `__durname, '..', ''Sia', ...`